Demo/Demo/SwiftUIComponents/SwiftUIPaymentButtonDemo.swift

```diff
@@ -0,0 +1,12 @@
+import SwiftUI
+
+
+@available(iOS 13.0.0, *)
```

Comment:
Do we get a warning without this? I thought we only supported iOS 13+ but maybe I am misremembering.

Reply:
i removed it and no warning!